src/main/java/com/revature/model/Comment.java

```diff
@@ -37,109 +34,68 @@
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private int id;
 	private Calendar date;
-	private String name;
 	private String text;
+	
 	@ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
-	@JoinColumn(name = "interview_comments")
-	private Interview interviewId;
-
+	@JoinColumn(name = "interview_interviewer")
+	

```

Comment:
Should there be a blank line here as well?

Reply:
Changes are made as per suggestion